packages/terra-clinical-result/translations/es-ES.json

```diff
@@ -2,6 +2,7 @@
   "Terra.clinicalResult.resultError": "Error",
   "Terra.clinicalResult.statusInError": "Con errores",
   "Terra.clinicalResult.statusInErrorHiddenText": "El valor no se ha documentado como Con errores",
+  "Terra.clinicalResult.statusInErrorStrikethrough": "Resultado {strikethroughResult}escrito Con errores",
```

Comment:
Looks like these are missing a space between `{strikethroughResult}` and `escrito`. Can you reach back out to the translations team to confirm if this is correct?

Reply:
Really good catch, thank you for calling that out. I left a comment on the Jira but I'm pretty positive you're correct, there should be a space. I went ahead and changed those translations, but can post another comment here with confirmation when they get back to me. [5057289](https://github.com/cerner/terra-clinical/pull/873/commits/505728911b13daacb552623d14b09e4c5fa31f9b)

Reply:
Translation team got back to me - there is supposed to be a space so we should be good to go now.